umocktypes/inc/c_pal/umocktypes_uuid_t.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef UMOCKTYPES_UUID_T_H
+#define UMOCKTYPES_UUID_T_H
+
+#include "ctest.h"
+
+#include "c_pal/uuid.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    typedef UUID_T* uuid_ptr;
+
+    void uuid_ptr_ToString(char* string, size_t bufferSize, const UUID_T* val);
+    int uuid_ptr_Compare(const UUID_T* left, const UUID_T* right);
+
+    void UUID_T_ToString(char* string, size_t bufferSize, const UUID_T val);
+    int UUID_T_Compare(const UUID_T left, const UUID_T right);
+
+    extern int umocktypes_UUID_T_register_types(void);

```

Comment:
>extern [](http://example.com/codeflow?start=4&length=6)

Needed?

Reply:
no, certainly not. extern is likely NEVER needed in C.

Reply:
removed all extern keywords from here.